README.md

```diff
@@ -31,7 +31,7 @@ These LLMs (Large Language Models) are all licensed for commercial use (e.g., Ap
 | phi-2 | 2023/12 | [phi-2 2.7B](https://huggingface.co/microsoft/phi-2) | [Microsoft](https://www.microsoft.com/en-us/research/blog/phi-2-the-surprising-power-of-small-language-models/) | 2.7 | [2048](https://huggingface.co/microsoft/phi-2/blob/main/config.json)| MIT | |
 | OLMo | 2024/02 | [OLMo 1B](https://huggingface.co/allenai/OLMo-1B), [OLMo 7B](https://huggingface.co/allenai/OLMo-7B), [OLMo 7B Twin 2T](https://huggingface.co/allenai/OLMo-7B-Twin-2T) | [AI2]([https://www.microsoft.com/en-us/research/blog/phi-2-the-surprising-power-of-small-language-models/](https://blog.allenai.org/hello-olmo-a-truly-open-llm-43f7e7359222)) | 1,7 | [2048](https://huggingface.co/allenai/OLMo-7B-Twin-2T/blob/main/config.json)| Apache 2.0 | |
 | Gemma | 2024/02 | [Gemma 7B](https://huggingface.co/google/gemma-7b), [Gemma 7B it](https://huggingface.co/google/gemma-7b-it), [Gemma 2B](https://huggingface.co/google/gemma-2b), [Gemma 2B it](https://huggingface.co/google/gemma-2b-it) | [Technical report](https://storage.googleapis.com/deepmind-media/gemma/gemma-report.pdf) | 2-7 | [8192](https://storage.googleapis.com/deepmind-media/gemma/gemma-report.pdf) | [Gemma Terms of Use](https://ai.google.dev/gemma/terms) | |
-
+| Zephyr  | 2024/24 | [Zephyr 7B](https://huggingface.co/HuggingFaceH4/zephyr-7b-gemma-v0.1 ), [Zephyr 7B it](https://huggingface.co/HuggingFaceH4/zephyr-7b-gemma-v0.1) | [Website](https://klu.ai/glossary/zephyr-7b) | 2-7 | [8192](https://huggingface.co/HuggingFaceH4/zephyr-7b-gemma-v0.1)| 	Apache 2.0 | |
```

Comment:
It seems the links for Zephyr 7B and Zephyr 7b it are the same? Is there only one model, or are one of the links wrong? Also, could you update the size from 2-7 to 7 please, since there's only the 7B variant available.

Reply:
yes thanks i will update